pkg/storage/sasNode.go

```diff
@@ -71,13 +71,10 @@ func (sas *sasStorage) NodePublishVolume(ctx context.Context, req *csi.NodePubli
 
 	klog.V(1).Infof("[START] publish volume (%s) wwn (%s) target (%s) lun (%s)", volumeName, wwn, req.GetTargetPath(), lun)
 
-	wwns := req.GetVolumeContext()[common.WWNs]
-	klog.Infof("SAS WWNS: %s", wwns)
-
 	// Initiate SAS attachment
 	klog.Info("initiating SAS connection...")
-	connector := saslib.Connector{Lun: lun, TargetWWNs: []string{wwns}}
-	path, err := saslib.Attach(ctx, connector, &saslib.OSioHandler{})
+	connector := saslib.Connector{TargetWWN: wwn}
```

Comment:
This should be VolumeWWN since your latest changes?

Reply:
Yes, I was waiting for csi-lib-sas changes, sorry about that. I have now committed the changes that match the latest sas library and I have also update go.mod/go.sum accordingly to the latest release tag.